### PR TITLE
Update calendar cell date slot text colors for lowest price highlighting

### PR DESCRIPTION
**Pull request by arrenleung**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #241

## Summary:

Updates the calendar cell date slot text to use `var(--ds-color-text-primary-default)` by default and `var(--ds-color-text-success-default)` when the slot is used with the `highlight` attribute

image

## Type of change:

- [X] New capability
- [X] Revision of an existing capability

## Checklist:

- [X] My update follows the CONTRIBUTING guidelines of this project
- [X] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style-auro-calendar-cell.scss [hunk 1]
```diff
@@ -122,7 +122,7 @@
 }
 
 ::slotted([slot^="date_"]) {
-  color: var(--ds-color-text-link-default, $ds-color-text-link-default);
+  color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
```
Comment (jordanjones243) [inline src/style-auro-calendar-cell.scss:125]: Per the design, we need to keep the token we are using for the primary use case of the date slot. Please revert this change. If there were instructions from a different designer, please let me know and I can discuss this with my team!

**Review by jordanjones243 — CHANGES_REQUESTED** [review]

File: src/style-auro-calendar-cell.scss (hunk 1 above)
Comment (jordanjones243) [reply src/style-auro-calendar-cell.scss:125]: The changes to the default color of the date price slot has been approved by designers on the Auro team.

**Review by jordanjones243 — APPROVED** [review]

**Comment by leeejune** [comment]
Auro design has confirmed changes to the default text color and highlighted text color. Thanks!

**Pull request merged**